tutorial/01_getting_started/README.md

```diff
@@ -119,7 +120,7 @@ With this in mind, let's change `render.gno` to add a simple "counter":
 - Add a new exported function `Add`, which increments this integer.
 - Write the value of this value when you call `Render`, importing `strconv` and
   using `strconv.Itoa` to convert it to a string.\
-  _Note: Gno doesn't currently support reflection, and as such package `fmt`._
+  _Note: Gno doesn't currently support reflection, and as such package `fmt`. But you can use them into the testing environment because Gno have temporary support for some extra packages like `fmt`, implemented with a "bridge" between go and gno._
```

Comment:
Can this "bridge" be detailed further or perhaps referenced here with a link to learn more?

Reply:
Ehhh, probably the most you can do to "reference" this into something more specific is pointing at [this code](https://github.com/gnolang/gno/blob/cfefb3b223de30a7db8b75c82c9853398caa2983/gnovm/tests/imports.go#L118). If you think that it's better than nothing, then we can add it...

Reply:
I think it makes sense to reference this for people as it brings transparency to the workaround we have proposed rather than stating that there is a bridge, we show how we are providing this functionality as well